### PR TITLE
Flag exceptions thrown by the ring-turnstile-mw

### PR DESCRIPTION
**Pull request by msprunck**
Exceptions coming from underlying middlewares can now be identified

### CODE REVIEW

File: src/ring/middleware/turnstile.clj [hunk 1]
```diff
@@ -131,21 +131,35 @@
...
+                                 (map #(rate-limit-headers % remaining-header-enabled))
+                                 (apply merge))
+                    response (try
+                               (handler request)
+                               (catch Exception e
+                                 (throw (ex-info (.getMessage e)
+                                                 {:origin :other-handlers}
+                                                 e))))]
+                (update response :headers merge headers)))))
+      (catch Exception e
```
Comment (yogsototh) [inline src/ring/middleware/turnstile.clj:158]: You should simply match on `ExceptionInfo` here.

**Review by yogsototh — APPROVED** [review]
I would use `ExceptionInfo` matching to narrow the catched exceptions here.

File: src/ring/middleware/turnstile.clj (hunk 1 above)
Comment (msprunck) [reply src/ring/middleware/turnstile.clj:158]: I can't, I don't know if the exception thrown by the dependencies used by the `ring-turnstile-middleware` are all wrapped in an `ExceptionInfo`

**Comment by hshahman** [comment]
No QA required

**Pull request merged**